Backend/lib/message-service/message-service.cabal

```diff
@@ -0,0 +1,125 @@
+cabal-version: 1.12
+
+-- This file has been generated from package.yaml by hpack version 0.35.2.
+--
+-- see: https://github.com/sol/hpack
+
+name:           message-service
+version:        0.1.0.0
+description:    Please see the README on GitHub at <https://github.com/nammayatri/nammayatri#readme>
+homepage:       https://github.com/“nammayatri/nammayatri”#readme
+bug-reports:    https://github.com/“nammayatri/nammayatri”/issues
+author:         “nammayatri”
+maintainer:     “nammayatri”
+copyright:      “2023 Juspay Technologies Private Limited”
+license:        AllRightsReserved
+build-type:     Simple
+
+source-repository head
+  type: git
+  location: https://github.com/“nammayatri/nammayatri”
+
+library
+  exposed-modules:
+      Lib.API.Dashboard.Message
```

Comment:
Why your lib uses `Lib` as base directory for all its files? It doesn't make sense

Reply:
@SanskarGubreley This comment is still not resolved and marked as resolved.